### DISCUSSION TITLE
Zustand Docs Imrovements: Add examples with Plain JS

### DISCUSSION THREAD

**Discussion by Diegogagan2587** [pmndrs/zustand#2683] [General] [answered]
## The Issue.
There are a lot of people who are starting to learn JavaScript or people who do not like TypeScript, many libraries and frameworks include examples for both in their docs, Including examples for only TypeScript will prevent, and or reduce the usage or increase the difficulty of understanding the library for many people.

See example below:
![image](https://github.com/user-attachments/assets/c742b501-77f5-4b74-ab81-ce1bd138bfea)

## Suggested Changes.
It would be great to include examples of usage using simple plain JavaScript in the docs instead of only explaining using TypeScript.

**Comment by dbritto-dev** [comment, marked as answer]
@Diegogagan2587 we're working on revamping docs -> https://github.com/pmndrs/zustand/pull/2508

**Reply by dbritto-dev** [reply]
I'll add JS and TS examples in the new docs
